test/appium/tests/atomic/chats/test_group_chat.py

```diff
@@ -3,20 +3,31 @@
 from tests.users import chat_users
 from views.sign_in_view import SignInView
 
+def return_left_chat_system_message(default_username):
+    return "*%s* left the group" % default_username
+
+def return_created_chat_system_message(username, chat_name):
+    return "*%s* created the group *%s*" % (username, chat_name)
+
+def return_joined_chat_system_message(default_username):
+    return "*%s* has joined the group" % default_username
 
 def create_users(driver_1, driver_2, username_1=None, username_2=None):
     device_1_sign_in, device_2_sign_in = SignInView(driver_1), SignInView(driver_2)
     if username_1 is not None and username_2 is not None:
         return device_1_sign_in.create_user(username_1), device_2_sign_in.create_user(username_1)
     return device_1_sign_in.create_user(), device_2_sign_in.create_user()
 
+def get_default_username(device_home):
+    device_profile_view = device_home.profile_button.click()
+    device_default_username = device_profile_view.default_username_text.text
+    device_home.home_button.click()
+    return device_default_username
 
-def create_and_join_group_chat(device_1_home, device_2_home, chat_name):
+def create_new_group_chat(device_1_home, device_2_home, chat_name):
     # device 2: get public key and default username
     device_2_public_key = device_2_home.get_public_key()
-    device_2_profile_view = device_2_home.profile_button.click()
-    device_2_default_username = device_2_profile_view.default_username_text.text
-    device_2_home.home_button.click()
+    device_2_default_username = get_default_username(device_2_home)
```

Comment:
we definitely don't need to do this action in all tests where are we going to create a group chat